### ISSUE TITLE
Cap bonus pp at a certain amount of plays

### ISSUE THREAD

**Issue by cl8n** [ppy/osu-performance#118]
idk if it's actually called "bonus pp" but what I'm talking about is this
https://github.com/ppy/osu-performance/blob/92b3eaf832f79eb3e0731c4ce75a8944a2e7b48f/src/performance/User.cpp#L57-L58

from what I've read previously, I understand this bonus to be for newer players to get a sense of progression via their pp/rank even when they aren't making top plays. if that's its only purpose, I think it's needlessly getting in the way of comparing players at higher levels because it doesn't reflect skill at all.

what I propose is for this function to be almost identical but with a flat cap at some reasonable number of scores you might achieve in a few months of active play

**Comment by stanriders** [comment]
you can say its already pretty much capped at ~7k scores
![image](https://user-images.githubusercontent.com/8269193/86907290-75a23b00-c11d-11ea-8240-1797512a5ee0.png)

**Comment by peppy** [comment]
It was implemented in a way where it naturally caps, as above. Are you saying 416 bonus is an issue?

**Comment by cl8n** [comment]
no, the amount doesn't matter that much, I just think it "caps" too late. plenty of experienced players are still in the 300-400 range of this bonus

hard to make any suggestions for numbers without data though. the only research I did before opening this was eyeballing random player's rank counts

**Comment by Magnus-Cosmos** [comment]
I think the current bonus is fine, since it can motivate players to try and set scores on more maps, rather than retrying the same few maps.

**Comment by cl8n** [comment]
I don't think that's the job of a performance rating, nor is it a practical way to gain pp right now anyway. someone who already plays the game a lot will only be able to take advantage of this bonus via a very slow grind that could easily be substituted by a few more top plays. I see your point, and I'd be one of the first to support good ways of motivating ppl to play a variety of maps too, but IMO this bonus is not doing much besides displacing players' ranks a bit based on nothing to do with skill

**Comment by peppy** [comment]
Actually it *is* the job of performance. That is the intention of the bonus. Else it wouldn't be there at all.

**Comment by WinterIsPretty** [comment]
I feel like for a more serious ranking system there would be no bonus at all but for user experience, I would say it's fine however 400 is a lot and it could boost you from rank 42340 with 5000 pp to rank 32340 with 5400 and different modes are similar except taiko if you are at the top any amount of pp can boost you a ton because of how flat the top players are now

**Comment by cl8n** [comment]
https://github.com/ppy/osu-queue-score-statistics/pull/205 at least temporarily resolved this for me with its 1000 scores cap. I can't tell if that part will be reverted though

**Comment by Rekunan** [comment]
It's intentional and there was a good discussion about it in #difficulty-osu: https://discord.com/channels/188630481301012481/380598781432823815/1201033630588338206

**Comment by peppy** [comment]
I think this can be closed for now.
